pytorch_lightning/accelerators/accelerator.py

```diff
@@ -351,26 +367,22 @@ def precision(self) -> int:
         return self.precision_plugin.precision
 
     @property
-    def scaler(self):
-        if hasattr(self.precision_plugin, "scaler"):
-            return self.precision_plugin.scaler
+    def scaler(self) -> Optional['GradScaler']:
 
-        return None
+        return getattr(self.precision_plugin, 'scaler', None)
```

Comment:
was this a bug?

Reply:
No, this behaves the same, but It get's around MyPys error "Precision Plugin has no attribute scaler", which we covered by returning None. But Mypy was not able to get this.